src/app/api/crud/groups.py

```diff
@@ -40,39 +40,55 @@ async def get_entity_group_id(table: Table, entry_id: int) -> int:
 
 async def _get_access_group_id(entry_id: int) -> int:
     entry = await crud.base.get_entry(accesses, entry_id)
+    if entry is None:
+        return None
     return entry["group_id"]
```

Comment:
Three things to consider : 
- if we keep this logic, the implementation is fine by me, we can also switch to a one-liner
```
return entry['group_id'] if isinstance(entry, dict) else None
```
- only a question but : shouldn't we raise an error if the entry-id doesn't exist?
- if we want to start optimizing SQL queries, here we actually only need the group_id field